akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -188,19 +192,15 @@ final class Flow[-In, +Out, +Mat](private[stream] override val module: Module)
   }
 
   /** INTERNAL API */
-  override private[stream] def andThen[U](op: StageModule): Repr[U, Mat] = {
+  // FIXME: Only exists to keep old stuff alive
+  private[stream] override def deprecatedAndThen[U](op: StageModule): Repr[U, Mat] = {
```

Comment:
stupid name, but this method will be removed as all stream-of-stream stages are migrated to the GraphStage infrastructure. `StageModule` and this method exist only to support them until removal.